src/software/world/robot_state.h

```diff
@@ -82,6 +126,9 @@ class RobotState
     Vector velocity_;
     Angle orientation_;
     AngularVelocity angular_velocity_;
+    bool ball_in_beam_;
```

Comment:
We should change the member variable name and argument names to `ball_in_mouth`?

Reply:
changed that